ORStools/gui/ORStoolsDialog.py

```diff
@@ -430,6 +430,7 @@ def _clear_listwidget(self) -> None:
                 if self.annotations:
                     self.project.annotationManager().removeAnnotation(self.annotations.pop(row))
                 self.routing_fromline_list.takeItem(row)
+                self.line_tool.error_idxs += 1
         else:
```

Comment:
This is a change that I think would lend itself nicely to red-green-testing:

The error description is "numbering of points gets messed up when adding points after having deleted points".

Thus, we could write a test that:
1. adds some points
2. deletes a point
3. adds another point
4. checks the numbering of the points.

We should see that test fail - so we commit, stating why it fails
We then fix the issue, as is done here
We should then see that test succeed - so we commit, stating what we fixed :)